### PR TITLE
Works about sections

### PR DESCRIPTION
**Pull request by letarulinda**
create anew branch works-section
create  about myself section
create ul and li for categories in works section.
create ul and li for categories in about myself section

### CODE REVIEW

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @letarulinda , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project in which the linter is not set up correctly. Please ensure that the Linter is set up correctly for this repository

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by letarulinda** [comment]
Hello Micheal,
Thanks for the comments, Is it possible to review my Code again, I have
Fixed the Linters Errors,
Thank you
Regards

**Comment by micheaol** [comment]
@letarulinda ,

Please re-submit from your dashboard for review.

File: index.css [hunk 1]
```diff
@@ -68,3 +69,199 @@ h1 {
...
+  margin-bottom: 12px;
+  background: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 16px;
+  width: max-content;
+}
+
+.about-section {
+  background: #f7f7f9;
+}
```
Comment (micheaol) [inline index.css]: Please ensure that the content of the `about section` is aligned with flexbox

File: index.html [hunk 2]
```diff
@@ -76,5 +76,180 @@ <h1 class="heading1">
...
+          About
+          <br />Myself
+        </h2>
+        <p class="content">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <p class="suggegiion-comment">LET’S CONNECT</p>
```
Comment (micheaol) [inline index.html:193]: Please ensure that the title (e.g., "Let's connect") uses a header tag `hx` according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @letarulinda , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by letarulinda** [comment]
The Icon is Inactive from the Dashboard.

**Comment by letarulinda** [comment]
Thanks, Micheal for the Comments, I have made the changes.

**Comment by Kakalanp** [comment]
Hello @letarulinda, please make sure you committed your code to the right branch so I can review it :pray:

**Comment by letarulinda** [comment]
Okay, let me do that

File: index.html [hunk 3]
```diff
@@ -46,7 +46,7 @@ <h1 class="heading1">
         you see and have a project you need coded, don’t hestiate to contact me.
       </p>
 
-      <p class="suggegiion-comment">LET’S CONNECT</p>
+      <header>class="suggegiion-comment">LET’S CONNECT</header>
```
Comment (Kakalanp) [inline index.html]: Looks like you confused `<h2>` with `<header>`, make sure to make this small change and your project will be approved! :wink:

**Review by Kakalanp — CHANGES_REQUESTED** [review]
# STATUS: Changes required :exclamation: 

### Hello there @letarulinda! hi

### Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. :memo: 

almost there

## Highlights :+1: 
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:
- Excellent communication with CRs :monocle_face: 

 ## Required Changes ♻️
***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (letarulinda) [reply index.html]: Noted,let me that.

**Comment by letarulinda** [comment]
Hello Joaquin,

I have made the changes

File: index.html (hunk 3 above)
Comment (letarulinda) [reply index.html]: Hello Kakalanp,I hope you are well.I made  the changes you requested me to do,My project was not approved.

**Review by ishimwezachee — APPROVED** [review]
Hi @letarulinda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**